components/scream/src/share/io/tests/io.cpp

```diff
@@ -262,6 +264,13 @@ std::shared_ptr<FieldRepository<Real>> get_test_repo(const Int num_lcols, const
       f4_host(ii,ipack)[ivec] = (ii) + (jj+1)/10.0;
     }
   }
+  // Update timestamp
+  util::TimeStamp time (0,0,0,0);
+  f1.get_header().get_tracking().update_time_stamp(time);
```

Comment:
If you want to init the ts of all the fields, you can simply do `repo->init_fields_time_stamp(time);`.